TLM/TLM/Manager/Impl/ExtSegmentEndManager.cs

```diff
@@ -170,7 +170,7 @@ public class ExtSegmentEndManager
 
         public ArrowDirection GetDirection(ushort segmentId0, ushort segmentId1, ushort nodeId = 0) {
             if (nodeId == 0) {
-                ref NetSegment netSegment = ref ((ushort)0).ToSegment();
+                ref NetSegment netSegment = ref segmentId0.ToSegment();
```

Comment:
lol where did ((ushort)0) came from?!

Reply:
![image](https://user-images.githubusercontent.com/26344691/175265217-9260e0a2-9cc5-4ead-b374-e88d14e939f6.png)
automated search and replace I suppose.

Reply:
I was the one who wrote the previous code.